tasks-by-users/Rehan-Khan-Engineer/08-Custom select box/style.css

```diff
@@ -0,0 +1,85 @@
+body {
+  margin: 0;
+  padding: 0;
+
+  box-sizing: border-box;
+
+  background-color: #004882;
+}
+
+.box {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.box select {
+  background-color: #0563af;
+  color: #fff;
+
+  height: 3rem;
+  width: 15rem;
+  padding: 10px;
+
+  font-size: 16px;
+
+  outline: none;
+  border: none;
+
+  /* Important */
+  -webkit-appearance: button;
+
+  box-shadow: 0 5px 25px rgba(0, 0, 0, 0.5);
+}
+
+/* Not Working */
+
+/* .box:before {
+  content: "f13a";
+  position: absolute;
+  top: 0;
+  right: 0;
+  height: 3rem;
+  width: 4rem;
+  background-color: #007ce0;
+  pointer-events: none;
+  text-align: center;
+  line-height: 3rem;
+  color: #fff;
+  font-size: 2rem;
+  font-family: "Font Awesome 5 Free";
+}
+
+.box:hover:before {
+  background-color: #0472ca;
+} */
+
+/* This is how i countered the issue */
```

Comment:
Remove these unnecessary comments because if you solved problem then its Great. you don't need to paste the problem in comment!!! @Rehan-Khan-Engineer

Reply:
I can understand your concern @Abbas4tech but i need it for myself in future. I am a slow learner.

Reply:
It's not about a learning speed. If you are raising a PR then it must be clear code so it can merge into `master` @Rehan-Khan-Engineer